### PR TITLE
Refactor frontend part 5

### PR DESCRIPTION
**Pull request by RichDom2185**
### Description

Minor changes, refer to commit history for details.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Code quality improvements

### Checklist

- [x] I have tested this code
- [ ] I have updated the documentation

### CODE REVIEW

File: src/commons/application/actions/SessionActions.ts [hunk 1]
```diff
@@ -152,6 +152,7 @@ export const setAdminPanelCourseRegistrations = (
 
 export const setGoogleUser = (user?: string) => action(SET_GOOGLE_USER, user);
 
+// TODO: Does not seem to be used anywhere, perhaps we can remove this?
 export const setGitHubAssessment = (missionRepoData: MissionRepoData) =>
   action(SET_GITHUB_ASSESSMENT, missionRepoData);
```
Comment (RichDom2185) [inline src/commons/application/actions/SessionActions.ts:157]: This action seems unused, shall we remove the action and relevant reducers/types?

File: src/features/academy/AcademyActions.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ import {
   SAVE_CANVAS
 } from './AcademyTypes';
 
+// TODO: Does not seem to be used anywhere, perhaps we can remove this?
 export const saveCanvas = (canvas: HTMLCanvasElement) => action(SAVE_CANVAS, canvas);
```
Comment (RichDom2185) [inline src/features/academy/AcademyActions.ts:14]: This action seems unused, shall we remove the action and relevant reducers/types?

**Review by RichDom2185 — COMMENTED** [review]
Can I get your opinions on whether we should remove these unused actions? @chownces @sayomaki

File: src/commons/application/actions/SessionActions.ts (hunk 1 above)
Comment (sayomaki) [reply src/commons/application/actions/SessionActions.ts:153]: The development and usage of GitHub Assessments does not seem to be very active at the moment, I think we can go ahead and remove this.

**Review by RichDom2185 — COMMENTED** [review]
Verbally agreed to merge. The remaining threads can always be resolved later.

**Pull request merged**